src/directives/internal/class.js

```diff
@@ -63,9 +63,11 @@ function setObjectClasses (el, obj) {
   var keys = Object.keys(obj)
   for (var i = 0, l = keys.length; i < l; i++) {
     var key = keys[i]
-    if (obj[key]) {
-      addClass(el, key)
-    }
+    if (!obj[key]) continue
+
+    key.split(' ').forEach(function (k) {
```

Comment:
That's what concerned me too, though I don't have an exact idea about the speed comparison between the check and a blind `split`.